src/components/layouts/default.tsx

```diff
@@ -1,11 +1,16 @@
 import { SiteHeader } from '../molecules/site-header/site-header';
+import { Footer } from '../molecules/footer/footer';
 import styles from './default.module.scss';
 
 export const DefaultLayout = ({ children }: any): JSX.Element => (
-  <div className={`${styles['grid-container']} ${styles['article-page']}`}>
-    <div className={styles['grid-header']}>
-      <SiteHeader />
+  <>
+    <div className={`${styles['grid-container']} ${styles['article-page']}`}>
+      <div className={styles['grid-header']}>
+        <SiteHeader />
+      </div>
+        {children}
     </div>
-    {children}
-  </div>
+    <Footer/>
```

Comment:
I wonder if `Footer` should be named `SiteFooter` to match `SiteHeader`?

Reply:
Agree. Will change it to SiteFooter